OpenRA.Mods.Common/Scripting/Global/BeaconGlobal.cs

```diff
@@ -0,0 +1,31 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System.Linq;
+using OpenRA.Mods.Common.Effects;
+using OpenRA.Scripting;
+
+namespace OpenRA.Mods.Common.Scripting
+{
+	[ScriptGlobal("Beacon")]
+	public class BeaconGlobal : ScriptGlobal
+	{
+		public BeaconGlobal(ScriptContext context) : base(context) { }
+
+		[Desc("Creates a new beacon that stays for the specified time at the specified WPos." +
+			"Does not remove player set beacons, nor gets removed by placing them.")]
+		public Beacon New(Player owner,  WPos position, int duration = 30 * 25, string palettePrefix = "player")
+		{
+			var playerBeacon = new Beacon(owner, position, duration, palettePrefix);
+			owner.PlayerActor.World.AddFrameEndTask(w => w.Add(playerBeacon));
```

Comment:
Hm, I should fix the radar ping, but do we want "press spacebar to go to beacon" (as there could be multiple beacons and just jumping to the last one sounds unpolished to me)?